deploy/olm-catalog/ibm-common-service-operator/3.3.0/ibm-common-service-operator.v3.3.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,155 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "operator.ibm.com/v3",
+          "kind": "CommonService",
+          "metadata": {
+            "name": "example-commonservice"
+          },
+          "spec": {
+            "size": 3
+          }
+        }
+      ]
+    capabilities: Basic Install
+    containerImage: quay.io/opencloudio/common-service-operator:3.3.0
+    createdAt: "2020-04-06T11:28:16Z"
+    description: The IBM Common Service Operator is used to deploy IBM Common Services
+    repository: https://github.com/IBM/ibm-common-service-operator
+    support: IBM
+  name: ibm-common-service-operator.v3.3.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: CommonService is the Schema for the commonservices API
+      displayName: CommonService
+      kind: CommonService
+      name: commonservices.operator.ibm.com
+      version: v3
+  description: |-
```

Comment:
I see the 3.4.0 has a new description format, for the 3.3.0 we can also follow this format for 3.3.0

Reply:
https://github.com/IBM/ibm-common-service-operator/blob/1c3fd959ab0002a68f4a770273a071a2b640bdff/deploy/olm-catalog/ibm-common-service-operator/3.4.0/ibm-common-service-operator.v3.4.0.clusterserviceversion.yaml#L40

Reply:
@DanielXLee 
Updated the description, please review again